Dockerfile

```diff
@@ -1,21 +1,33 @@
-FROM node:20 as build
+ARG CONFIG_UI_TAG_OR_HASH=265e3598b17417ed4becbfc067fe12ec2fcbf665
```

Comment:
Shouldn't we set the latest tag? Chaking out to master?

Reply:
we dont have releases for now there, so for now i think this is for the best